### PR TITLE
Linear models

### PR DESCRIPTION
**Pull request by MikeLippincott**
Look deeper into the visualization of linear models of single cell morphology features.

### CODE REVIEW

File: 3.sc_Morphology_Linear_Modeling/README.md [hunk 1]
```diff
@@ -20,31 +20,46 @@ and  files
 `2c.Linear_Modeling_Visualization.ipynb`
 `2d.Linear_Modeling_Visualization.ipynb`
 
-visualize each model into one pdf
+to visualize each model into one pdf
 
-# The Models:
+#### Only for the single beta + cell count models:
+I have included a visualization from [CFReT Analysis](https://github.com/gwaybio/CFReT_data/blob/368f0baf7b027f5614de7173bbc7336add81fcb3/4.analyze-data/notebooks/linear_model/1.visualize_linear_model.ipynb) to look at the beta coefficients across morphology feature types and cellular compartments:
```
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/README.md:26]:
```suggestion
I have included a visualization from [CFReT Analysis](https://github.com/gwaybio/CFReT_data/blob/368f0baf7b027f5614de7173bbc7336add81fcb3/4.analyze-data/notebooks/linear_model/1.visualize_linear_model.ipynb) to visluaize the beta coefficients across morphology feature types and cellular compartments:
```

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r [hunk 2]
```diff
@@ -66,18 +69,16 @@ lm_df$channel_learned <- dplyr::recode(lm_df$channel_cleaned,
...
 lm_df$abs_twob_Metadata_Treatment_Inhibitor_Dose <- abs(lm_df$twob_Metadata_Treatment_Inhibitor_Dose)
 lm_df$abs_Treatment_Dose <- abs(lm_df$Treatment_Dose)
 
-head(lm_df)
-
 loop_list <- unique(lm_df$inducer1_inhibitor_inhibitor_dose__inducer1_dose)
 x_list <- c('abs_twob_Metadata_Treatment_Inhibitor_Dose','abs_Treatment_Dose')
 
-pdf(file=lm_cp_fig )
+pdf(file=lm_cp_fig_abs )
```
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:81]: Consider adding some documentation here.

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r [hunk 3]
```diff
@@ -98,4 +99,81 @@ for (i in 1:length(loop_list)){
...
+    plot(lm_fig_gg)
+    }
+}
+dev.off()
+
+pdf(file=lm_cp_fig)
+for (i in 1:length(loop_list)){
+    df <- lm_df[lm_df$dosage_treatments_list == loop_list[i],]
+    lm_fig_gg <- (
+        ggplot(df, aes(x = oneb_Metadata_Treatment_Dose_Inhibitor_Dose, y = Metadata_number_of_singlecells))
```
Comment (axiomcura) [inline 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:134]: same comment above

**Review by axiomcura — APPROVED** [review]
LGTM, added some suggestions. 

I've noticed that the R scripts are primarily meant for plotting; however, it is difficult to follow what is being plotted. I do not know if adding some documentation about the plot is necessary. Furthermore, I'll leave that up to you! Otherwise, great work!

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r (hunk 2 above)
Comment (MikeLippincott) [reply 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:81]: I might remove this in the future, I am holding off on extra changes for now

File: 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r (hunk 3 above)
Comment (MikeLippincott) [reply 3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r:134]: same here, not a strong analysis

**Pull request merged**